app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  before_filter :authenticate_user!
+
+  def show
+    @user = User.find(params[:id])
```

Comment:
![screen shot 2015-06-15 at 12 58 53 am](https://cloud.githubusercontent.com/assets/2347787/8152914/cdfb327a-12f9-11e5-8b1b-affe0387dca8.png)

Have a look at the `before_action` at the top and the `set_account` under the protected part at the bottom.